src/qutip_qip/circuit/circuit.py

```diff
@@ -940,84 +947,77 @@ def compute_unitary(self):
 
         # This slightly convoluted dance with the conversion formats is because
 
-    # image conversion has optional dependencies.  We always want the `png` and
-    # `svg` methods to be available so that they are discoverable by the user,
-    # however if one is called without the required dependency, then they'll
-    # get a `RuntimeError` explaining the problem.  We only want the IPython
-    # magic methods `_repr_xxx_` to be defined if we know that the image
-    # conversion is available, so the user doesn't get exceptions on display
-    # because IPython tried to do something behind their back.
-
-    def _generate_png(self):
-        return TeXRenderer(self).raw_img(file_type="png")
-
-    def _generate_svg(self):
-        return TeXRenderer(self).raw_img(file_type="svg")
-
-    @property
-    def png(self):
-        """
-        Return the png file
-        """
-        return DisplayImage(self._repr_png_(), embed=True)
-
-    @property
-    def svg(self):
-        """
-        Return the svg file
-        """
-        return DisplaySVG(self._repr_svg_())
-
     def draw(
         self,
         renderer="matplotlib",
         file_type="png",
         dpi=None,
-        file_name="exported_pic",
-        file_path="",
+        file_path="circuit",
+        save=False,
         **kwargs,
     ):
         """
         Export circuit object as an image file in a supported format.
 
         Parameters
         ----------
-        renderer : choose the renderer for the circuit.
-                   Default: 'matplotlib'
+        renderer : str, optional
+            The renderer to use for the circuit. Options are 'latex', 'matplotlib', or 'text'.
+            Default is 'matplotlib'.
+
+        file_type : str, optional
+            The type of the image file to export. Supported types are 'svg' and 'png'.
+            Default is 'png'.
 
-        file_type : Provide a supported image file_type eg: "svg"/"png".
-                   Default : "png".
+        dpi : int, optional
+            The image density in dots per inch (dpi). Applicable for PNG, not for SVG.
+            Default is None (set to 100 internally for PNG).
 
-        dpi : Image density in Dots per inch(dpi)
-            Applicable for PNG, NA for SVG.
-            Default : None, though it's set to 100 internally for PNG
+        file_path : str, optional
+            The path to save the image file. Default is the current working directory.
 
-        file_name : Filename of the exported image.
-                    Default : "exported_pic"
+        save : bool, optional
+            If True, the image will be saved to the specified path.
+            Default is False.
 
-        file_path : Path to which the file has to be exported.
-                    Default : ""
-                    Note : User should have write access to the location.
+        kwargs : dict
+            Additional keyword arguments passed to the renderer.
```

Comment:
Need to point to the doc where all the kwargs  are listed.

Reply:
All the kwarg arguments are mentioned in `base_renderer.py`'s `StyleConfig Dataclass`, I have added a note.